admin/index.php

```diff
@@ -29,6 +50,7 @@
 <div id="wrapper" style="overflow-y: auto;">
 	<div class="admin-panel">
 		<h2><a class="back-to-pb" href="../">Photobooth</a></h2>
+		<?php if( !$config['login_enabled'] || (isset($_SESSION['login']) && $_SESSION['login'] == $hash)): ?>

```

Comment:
I would prefer to use a dedicated login page. Makes things cleaner and allows to add authentication easily to other pages. Than move the Auth test to a single file which will redirect the user to the login page.